apps/www/components/layout/navbar.tsx

```diff
@@ -42,17 +44,6 @@ export function NavBar({
         <div className="flex items-center space-x-3">
           {rightElements}
 
-          {/* {!user ? (
-              <Link
-                href="/login"
-                className={cn(
-                  buttonVariants({ variant: "outline", size: "sm" })
```

Comment:
We need a button. Right now there is nothing there for the user when they try to sign in.
![image](https://github.com/projectx-codehagen/projectx/assets/24507211/743db632-6fb4-4b6c-9125-15024294276f)

It should be like this:
![image](https://github.com/projectx-codehagen/projectx/assets/24507211/cd8ed464-2e57-450a-9695-27c0ffc70622)

Reply:
You're right! I must have missed it. I'll fix this